specification/metadata/metadata_schema.md

```diff
@@ -0,0 +1,75 @@
+# FOCUS Metadata Schema
+The FOCUS metadata schema provides a list of the columns present in the FOCUS dataset provided.   
+
+
+## Logical Section: Schema
+
+## Available Meta Data
+ * ### schema_id
+   * Use: unique identifier for the schema
+   * Type: STRING
+ * ### index
+   * use: provides the index number for the column to specify the order of the columns, 0 based
+   * type: INTEGER
+ * ### column_name
+   * use: provides the date the schema was created
+   * type: STRING
+ * ### data_type (provided by API delivery mechanism only)
```

Comment:
Once I got here, I realized we should probably use an industry standard schema format rather than defining our own. Thoughts? JSON Schema should work, right?

Reply:
As discussed today we are choosing to delay this till a future version. Current content specifies what metadata we want.